front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -183,6 +191,51 @@ export default function AssistantNew({
     setGreeting(getRandomGreetingForName(user.firstName));
   }, [user]);
 
+  function StartHelperConversationButton({
```

Comment:
Can we move this component outside of this component?

Reply:
I don't think `StartHelperConversationButton` as is made sense outside of the component -> It needs access to the state, and is used only in here. 

As discussed IRL, what I rather do is kill the component, duplicate the button and only have the submit function shared in the component. 

I added a commit for that, let me know if that works better for you!